src/rollup.ts

```diff
@@ -66,7 +66,7 @@ export async function rollupClient(
       nodeResolve({resolveOnly: BUNDLED_MODULES}),
       importResolve(input, root, path),
       esbuild({
-        target: "es2022",
+        target: ["es2022", "chrome96", "firefox96", "safari16", "node18"],
```

Comment:
These are the latest browsers as of January 2022, with the exception of Safari which shipped September 2022.